tiledb/sm/serialization/query.cc

```diff
@@ -434,6 +474,50 @@ Status reader_to_capnp(
   // Read state
   RETURN_NOT_OK(read_state_to_capnp(array_schema, reader, reader_builder));
 
+  const QueryCondition* condition = reader.condition();
+  if (!condition->empty()) {
```

Comment:
@Shelnutt2 This is the line that prevents us from pathing to `condition_to_capnp` and finding empty clauses or clause combination ops.

Reply:
Thanks!